subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CopyPermissionsIntegrationTest.groovy

```diff
@@ -87,12 +87,14 @@ class CopyPermissionsIntegrationTest extends AbstractIntegrationSpec implements
              }
             """
         when:
+        executer.noDeprecationChecks()
```

Comment:
Shouldn't these be using `expectDocumentedDeprecationWarning` instead?

Reply:
It might very well be, I will look into it. (Atm this PR is only for saving previous work removed from the original PR.)

Reply:
Fixed.